frontend/src/pages/ResetPassword.jsx

```
@@ -0,0 +1,64 @@
+          required
+          placeholder="Enter new password"
+          value={password}
+          onChange={(e) => setPassword(e.target.value)}
+          className="w-full p-2 border rounded mb-4"
+        />
+        <button
+          type="submit"
+          className="w-full bg-green-600 text-white py-2 px-4 rounded hover:bg-green-700"
+        >
+            {loading ? (
+              <div className="h-5 w-5 border-2 border-white border-t-transparent rounded-full animate-spin mx-auto"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)